docs/Installation.md

```diff
@@ -700,20 +667,22 @@ sudo apt install postgresql libdbd-pg-perl
 
 > **Note:** See the [backend configuration] documentation for details.
 
-Initialize Zonemaster database and user (unless you keep an old database):
+To create the database and the database user (unless you keep an old database)
+run the command on one line.
 
 ```sh
-sudo -u postgres createuser -P zonemaster
+sudo -u postgres psql -c "CREATE USER zonemaster WITH PASSWORD 'zonemaster';"  -c "CREATE DATABASE zonemaster WITH ENCODING 'UTF8';"

```

Comment:
same remark and suggestion as with CentOS

Reply:
Same question on sudo.